src/Request/BaseRequest.php

```diff
@@ -103,7 +103,7 @@ public function sendRequest()
     protected function requestOptions()
     {
         $opts = [
-            'verify' => __DIR__ . '/../../resources/cert.pem',
+            'verify' => true,
```

Comment:
Maybe this should be a config setting passed to the SDK? "Use this cert path instead."